tools/cli/main.js

```diff
@@ -592,7 +592,7 @@ Fiber(function () {
 
   // Check required Node version.
   // This code is duplicated in tools/server/boot.js.
-  var MIN_NODE_VERSION = 'v0.10.41';
+  var MIN_NODE_VERSION = 'v8.0.0';
```

Comment:
Not requiring Node 8.9.3 here because that could break non-Galaxy deployment environments that aren't yet on the very latest version of Node 8, which is not ideal but definitely acceptable.

Reply:
Hmm, not clear to me in what context tools/cli/main.js gets run by any version of Node other than the one meteor chooses...